rclc/src/rclc/executor.c

```diff
@@ -906,6 +1143,177 @@ _rclc_take_new_data(rclc_executor_handle_t * handle, rcl_wait_set_t * wait_set)
       // notification, that timer is ready already done in _rclc_evaluate_data_availability()
       break;
 
+    case ACTION_CLIENT:
+      if (handle->action_client->goal_response_available) {
+        Generic_SendGoal_Response aux_goal_response;
+        rmw_request_id_t aux_goal_response_header;
+        rc = rcl_action_take_goal_response(
+          &handle->action_client->rcl_handle,
+          &aux_goal_response_header,
+          &aux_goal_response);
+        if (rc != RCL_RET_OK) {
+          PRINT_RCLC_ERROR(rclc_take_new_data, rcl_action_take_goal_response);
+          RCUTILS_LOG_ERROR_NAMED(ROS_PACKAGE_NAME, "Error number: %d", rc);
+          return rc;
+        }
+        rclc_action_goal_handle_t * goal_handle =
+          rclc_action_get_handle_by_goal_request_sequence_number(
+          handle->action_client, aux_goal_response_header.sequence_number);
+        if (NULL != goal_handle) {
+          goal_handle->available_goal_response = true;
+          goal_handle->goal_accepted = aux_goal_response.accepted;
+        }
+      }
+      if (handle->action_client->feedback_available) {
+        rc = rcl_action_take_feedback(
+          &handle->action_client->rcl_handle,
+          handle->action_client->ros_feedback);
+
+        if (rc != RCL_RET_OK) {
```

Comment:
```suggestion
          handle->action_client->ros_feedback);
        if (rc != RCL_RET_OK) {
```
For consistency with above code layout.

Reply:
1e4b32f7207225b8456aa3f32dfdb43846cf1fbd